nodejs/week3/src/warmUp3/index3.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const app = express();
+const PORT = 3000;
+
+app.use(express.json());
+app.use(
+  express.urlencoded({
+    extended: true,
+  })
+);
+
+// Calculator using req.body
+
+app.get("/calculator", (req, res) => {
```

Comment:
I think the idea was that you would always have these paths:

/calculator/add
/calculator/multiply
etc...

and you would send the things to add/multiply/etc... via query params or the body of a request. 

In this example you're sending the method itself as part of the body. But GET requests don't have a body, you'll often use POST and PUT requests to send data via the body of the request.